.changes/next-release/bugfix-AWSSDKforJavav2-4ab9305.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "AWS SDK for Java v2", 
+    "type": "bugfix", 
+    "description": "Allows the read and write timeout to be 0"
```

Comment:
The category should be `Netty NIO Http Client` since that's why the change was made.  The type should be `feature`.

Can we also link the issue in the description? Something like the following:
```
 Allow customers to disable read and write timeout by setting `Duration.ZERO` to `readTimeout` and `writeTimeout`. See [#1281](https://github.com/aws/aws-sdk-java-v2/issues/1281)
```